rednotebook/storage.py

```diff
@@ -135,6 +135,10 @@ def get_filename(infix):
         yaml.dump(content, f, Dumper=Dumper, allow_unicode=True)
 
     if os.path.exists(filename):
+        if os.path.getmtime(filename)!=month.mtime:
```

Comment:
Can you store the result in a local mtime variable to avoid calling getmtime twice?